addons/web/static/src/views/fields/relational_utils.js

```diff
@@ -639,6 +640,7 @@ export function useOpenX2ManyRecord({
         const form = await getFormViewInfo({ list, activeField, viewService, userService, env });
 
         let deleteRecord;
+        let button_text = "";
```

Comment:
Would set it to undefined and not empty text, it could possibly mean that we don't want a label is that case, which is strange.